src/templater.rs

```diff
@@ -526,14 +527,27 @@ impl<'a> FormattedCommitOrChangeId<'a> {
             format!("{prefix}[{rest}]")
         }
     }
+
+    fn format_short_styled_prefix(&self, formatter: &mut dyn Formatter) -> io::Result<()> {
+        let (prefix, rest) = extract_entire_prefix_and_trimmed_tail(
+            self.hex(),
+            self.repo.shortest_unique_id_prefix_len(self.as_bytes()),
+            self.trim_display_to,
+        );
+        formatter.with_label("id", |fmt| {
```

Comment:
nit: if we label it as `id`, perhaps the default and bracket ones should also be labeled as `id`.

Personally I prefer not adding `id` in addition to `commit_id`/`change_id`.

Reply:
Good point. There's no need to add the `id` label here, I think.

Reply:
The `id` makes the config inside `colors.toml` slightly easier later. I can duplicate each of those lines for `commit_id` and `change_id` if you prefer. Or I can add the `id` label to the other formats.

Reply:
Ah, I see. Yes, seems fine to duplicate a bit there. Or maybe you can use `short` where you currently use `id` in `colors.toml`.